stable/traefik/templates/service.yaml

```diff
@@ -55,7 +55,7 @@ spec:
     targetPort: httpn
     {{- end }}
   {{- if (and (.Values.metrics.prometheus.enabled) (not (.Values.metrics.prometheus.restrictAccess)))}}
-  - port: 8080
```

Comment:
I am not person eligible for the review, but I wanted to throw my 3 cents. Shouldn't this be a completely separate service? I would like to keep metrics internal for the cluster.

Reply:
I vote for this. I can separate the `Service`

Reply:
Aren't all ports exported to the external ELB? I believe they were, when I tried to use it last time.

Reply:
I've doubled check this and actually all the ports are exposed to the ELB. I'll make another round of changes and wire a PR for today